portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -23,12 +23,14 @@
   private String name;
   private String comment;
   private long timestamp;
+  private double score;
 
-  public Comment(long id, String name, String comment, long timestamp) {
+  public Comment(long id, String name, String comment, long timestamp, double score) {
```

Comment:
nit: you might want to add comment or javadoc to describe each param, e.g. what specific score it refers to? what does it mean when the score is high and when it is low? what is the valid range of the score?